build.sh

```diff
@@ -1,73 +1,77 @@
 #!/bin/bash
-#
-#
 
-RELEASE_VERSION=
-buildpath=
+set -e
 
-function setuptree() {
-  b=$( mktemp -d $buildpath/gh-ostXXXXXX ) || return 1
-  mkdir -p $b/gh-ost
-  mkdir -p $b/gh-ost/usr/bin
-  echo $b
-}
-
-function build {
-  osname=$1
-  osshort=$2
-  GOOS=$3
-  GOARCH=$4
+. ./script/common
+buildpath=build
+timestamp=$(date "+%Y%m%d%H%M%S")
 
+function build() {
+  goos=$1
+  osshort=$goos
 
+  envs="-e GOOS=$goos \
+    -e GOARCH=amd64 \
```

Comment:
Technically, nothing is stopping this from extending to also offering `386` or windows at this point. But I didn't go that far.

Reply:
There's an open issue+PR for supporting Raspberry Pi. Truly.

Reply:
Hah. I haven't tried compiling for ARM but it doesn't look that difficult https://www.thepolyglotdeveloper.com/2017/04/cross-compiling-golang-applications-raspberry-pi/

I imagine ARM is going to get more popular with the Intel NUC crowd and the rumors MacBook Pro will ditch Intel and move to it.